### ISSUE TITLE
MAX value for Unsigned8

### ISSUE THREAD

**Issue by hallo02** [empeeoh/BACnet4J#3]
Hey

In com.serotonin.bacnet4j.type.primitive.Unsigned8, Line 32
`private static final int MAX = 0xff - 1;`

This should be 0xff, isn't it? 
8 bits, 2^8 = 256 possibilities,
so, 2^8 -1 = 255 = MAX Value = 0xff

**Comment by mlohbihler** [comment]
Yes, you are correct. A similar fix was made in Unsigned32. Unsigned16 looks to already be correct (although it may have been fixed recently). Thanks for pointing this out.
